manifests/config.pp

```diff
@@ -61,11 +61,6 @@
 
     'none'     : {
       # Don't configure apache, gunicorn or nginx. Leave all webserver configuration to something external.
```

Comment:
This will not work. Even if we don't want the module to configure the webserver. We need a user for the file/directory permissions. Eg: If you want to configure your own apache, you still have to set the apache user, so the files can be accessed.